src/integration-tests/test_admin_client.py

```diff
@@ -96,7 +102,96 @@ def test_breathing(single_node: Cluster):
     admin.stop()
 
 
-def test_purge_breathing(single_node: Cluster):
+def test_admin_encoding(single_node: Cluster) -> None:
+    """
+    Test: admin commands output format.
+    Preconditions:
+    - Establish admin session with the cluster.
+
+    Stage 1:
+    - Send json and plaintext admin commands with 'TEXT' output encoding.
+    - Expect the received responses in text format.
+
+    Stage 2:
+    - Send json and plaintext admin commands with 'JSON_COMPACT' output encoding.
+    - Expect the received responses in compact json format.
+
+    Stage 3:
+    - Send json and plaintext admin commands with 'JSON_PRETTY' output encoding.
+    - Expect the received responses in pretty json format.
+
+    Stage 4:
+    - Send json and plaintext admin commands with incorrect output encoding.
+    - Expect the received responses with decode error message.
+
+    Stage 5:
+    - Send json and plaintext admin commands without output encoding option.
+    - Expect the received responses in text format.
+
+    Concerns:
+    - It's possible to pass output encoding option to both json and plaintext encoded admin commands.
+    - TEXT, JSON_COMPACT, JSON_PRETTY encoding formats are correctly supported.
+    - Commands with incorrect encoding are handled with decode error.
+    - Commands without encoding return text output for backward compatibility.
+    """
+    host, port = get_endpoint(single_node)
+
+    def is_compact(json_str: str) -> bool:
+        return "    " not in json_str
+
+    # Start the admin client
+    admin = AdminClient()
+    admin.connect(host, port)
+
+    # Stage 1: encode as TEXT
+    cmds = [json.dumps({"help": {}, "encoding": "TEXT"}), "ENCODING TEXT HELP"]
+    for cmd in cmds:
+        res = admin.send_admin(cmd)
+        assert "This process responds to the following CMD subcommands:" in res
+
+    # Stage 2: encode as JSON_COMPACT
+    cmds = [
+        json.dumps({"help": {}, "encoding": "JSON_COMPACT"}),
+        "ENCODING JSON_COMPACT HELP",
```

Comment:
Is it okay to use underscores in plaintext admin commands?
Why do we avoid using underscores in all cmds, like `BROKERCONFIG DUMP`?